src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,16 +1,24 @@
 package mate.academy.service;
 
 public class AuthenticationService {
+    private static final UserService userService = new UserService();
+
     /**
      * Imagine that some user wants to login to your site.
      * You should check if user credentials (login and password) are valid or not.
      * All users are stored in <code>UserService</code> class.
-     * @param email - user's email
+     *
+     * @param email    - user's email
      * @param password - user's password
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        if (userService.findByEmail(email) == null) {
+            return false;
+        } else {
+            return userService.findByEmail(email).getPassword().equals(password);
+
+        }
```

Comment:
in this case, you call the findByEmail method twice with the same parameters. I think it is better to create a local change, because I can inject it into performance